src/Vlingo.Common/Message/AsyncMessageQueue.cs

```diff
@@ -121,7 +118,34 @@ public virtual void Run()
             finally
             {
                 dispatching.Set(false);
+                endDispatchingEvent.Set();
+            }
+        }
+        
+        public void Dispose()
+        {
+            Dispose(true);
+            GC.SuppressFinalize(this);
+        }
+        
+        protected virtual void Dispose(bool disposing)
+        {
+            if (disposed)
```

Comment:
We have almost always used `{..}`, even for a single line statement. Would be nice to have it consistent.

Reply:
Ok, didn't notice the change. I'll fix it.